tutorials/preprocessing/plot_70_fnirs_processing.py

```diff
@@ -0,0 +1,199 @@
+"""
+.. _tut-fnirs-processing:
+
+Preprocessing fNIRS data
+========================
+
+This tutorial covers how to convert fNIRS data from raw measurements to
+relative oxyhaemoglobin (HbO) and deoxyhaemoglobin (HbR) concentration.
+
+.. contents:: Page contents
+   :local:
+   :depth: 2
+
+Here we will work with the :ref:`fNIRS motor data <fnirs-motor-dataset>`.
+"""
+# sphinx_gallery_thumbnail_number = 3
+
+import os
+import mne
+from itertools import compress
+import matplotlib.pyplot as plt
+from mne.io.pick import _picks_to_idx
+
+
+fnirs_data_folder = mne.datasets.fnirs_motor.data_path()
+fnirs_raw_dir = os.path.join(fnirs_data_folder, 'Participant-1')
+raw_intensity = mne.io.read_raw_nirx(fnirs_raw_dir, verbose=True).load_data()
+
+
+###############################################################################
+# Selecting channels appropriate for detecting neural responses
+# -------------------------------------------------------------
+#
+# First we select optodes that are sufficiently separated to detect neural
+# responses (approximately 3cm) for further processing.
+
+long_channels = ~mne.preprocessing._beer_lambert_law._short_channels(
+    raw_intensity)
+raw_intensity.pick(mne.pick_channels(raw_intensity.ch_names,
+                                     list(compress(raw_intensity.ch_names,
+                                                   long_channels))))
+raw_intensity.plot(n_channels=len(raw_intensity.ch_names), duration=500)
+
+
+###############################################################################
+# Converting from raw intensity to optical density
+# ------------------------------------------------
+#
+# The raw intensity values are then converted to optical density.
+
+raw_od = mne.preprocessing.optical_density(raw_intensity)
+raw_od.plot(n_channels=len(raw_od.ch_names), duration=500)
+
+
+###############################################################################
+# Converting from optical density to haemoglobin
+# ----------------------------------------------
+#
+# Next we convert the optical density data to haemoglobin concentration using
+# the modified Beer-Lambert law.
+
+raw_fnirs = mne.preprocessing.beer_lambert_law(raw_od)
+raw_fnirs.plot(n_channels=len(raw_fnirs.ch_names), duration=500)
+
+
+###############################################################################
+# Remove heart rate from signal
+# -----------------------------
+#
+# The haemodynamic response has frequency content predominantly below 0.5 Hz.
+# An increase in activity around 1 Hz can be seen that is due to the heart beat
+# and is unwanted. So we use low pass filter can be used to remove this.
+
+raw_fnirs.plot_psd(average=True)
```

Comment:
```suggestion
fig = raw_fnirs.plot_psd(average=True)
```
and then something like `fig.suptitle('Before filtering', weight='bold', size='xx-large')`

same for second plot, with title "After filtering"